nodes/ProgramNode.java

```diff
@@ -0,0 +1,57 @@
+package nodes;
+
+import provided.JottTree;
+import provided.Token;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class ProgramNode implements JottTree {
+
+    private ArrayList<JottTree> children = new ArrayList<>();
```

Comment:
In this case, that makes sense. In other Nodes where children can be different node types, it would have to be `ArrayList<JottTree>` afaik. I did this for consistency between Node classes, but it's not necessary, so I could change it if wanted.

Reply:
Made said change